content/news/XXXX-XX-XX-mixxx-2-3-0-release.md

```diff
@@ -0,0 +1,126 @@
+title: Mixxx 2.3.0 released
+authors: Jan Holthuis
+tags: 2.3, 2.3.0, release announcement
+status: draft
+comments: yes
+
+After a long waiting time, we are proud to announce the release of Mixxx 2.3.0, [download it!]({filename}/pages/download.md#stable)
+This release brings [hotcue colors & labels]({filename}/news/2020-08-25-new-in-2-3-hotcue-colors.md), import of [Serato](2021-02-08-new-in-2-3-serato-support.md) and [Rekordbox]({filename}/news/2020-07-20-new-in-2-3-rekordbox-support.md) libraries, [more accurate key detection]({filename}/news/2021-04-08-new-in-2-3-keyfinder.md), [intro/outro sections](2020-07-09-intro-outro-sections.md), a revamped LateNight skin, multithreaded analysis, [deck cloning]({filename}/news/2020-11-15-new-in-2-3-deck-clone.md), [track color support]({filename}/news/2020-10-24-new-in-2-3-track-colors.md) and more.
+
+![Screenshot of a Mixxx deck with the intro and outro cues and hotcue editing menu]({static}/images/news/screenshot-2.3-beta-deck.png){: style="max-height: 275px" }
+
+This has been a rather big release, with lots of useful changes, bug fixes and improvements.
+Almost 100 people contributed a total of 7183 changes.
+All in all, a whopping 1 million lines of code were changed since the [2.2.4 release]({filename}/news/2020-06-11-Mixxx-2-2-4-released.md)!
+
+**TODO:** Update the numbers above using `git shortlog -sn release-2.2.4..upstream/2.3 | wc -l`, `git log --oneline --no-merges release-2.2.4..upstream/2.3 -- . ':!lib' | wc -l` and `git diff --shortstat release-2.2.4..upstream/2.3 -- . ':!lib`.
+
+Thanks to all the developers and contributors, we really appreciate it!
+
+## Changelog
+
+**TODO**
+
+## Credits
+
+Many people spent their free time working on Mixxx.
+Thanks a lot to all of them:
+
+    $ git shortlog -sn release-2.2.4..upstream/2.3
+    2305  Jan Holthuis
```

Comment:
and we do have a winner : )
:+1: 

maybe we should mention the numbers stand for commits (just guessing),
not for drunk coffees, hours spent with c++ or how often the person started Mixxx..

Reply:
I can also check if there is a way to list committers without commit numbers. I just thought it would be a nice idea to list everyone who worked on this release to incentivize contributing :D

Reply:
In general such thing is s good idea.
It "just" forgets the translators and manual authors and other work that does not lead to a countable commit.

Reply:
even though I dislike score boards // peolpe assuming bigger numbers are better...
I'm not questioning the list. It is a nice and simple way it illustrates how much work was done  (if people know what a commit is)